packages/button/src/Button.tsx

```diff
@@ -1,17 +1,21 @@
 import { Slot } from '@radix-ui/react-slot';
 import { clsx as cx } from 'clsx';
-import {
-  type CSSProperties,
-  type ComponentProps,
-  type ReactNode,
-  forwardRef,
-} from 'react';
-import { match } from 'ts-pattern';
-import styles from './Button.module.css';
-
-type ButtonColor = 'primary' | 'black' | 'white';
-
-type ButtonVariant = 'filled' | 'outline' | 'weak';
+import { type ComponentProps, type ReactNode, forwardRef } from 'react';
+import * as styles from './Button.css';
+
+export const ButtonColor = {
+  primary: 'primary',
+  black: 'black',
+  white: 'white',
+} as const;
+export type ButtonColor = (typeof ButtonColor)[keyof typeof ButtonColor];
+
+export const ButtonVariant = {
+  filled: 'filled',
+  outline: 'outline',
+  weak: 'weak',
+} as const;
+export type ButtonVariant = (typeof ButtonVariant)[keyof typeof ButtonVariant];
 
 export interface ButtonProps extends ComponentProps<'button'> {
```

Comment:
```suggestion
export interface ButtonProps extends ComponentProps<'button'> {
  color?: ButtonColor;
  variant?: ButtonVariant;
  asChild?: boolean;
}
```
기본 props들은 명시안해줘도 되지않았을까 싶습니다.

Reply:
아하 그러게.. 기존에는 왜 정의한걸까 ㅋㅋ

Reply:
done! [ab73538](https://github.com/sipe-team/side/pull/150/commits/ab73538b9753550672b3c4eb3cdfa8bf9d1e7dce)